lib/sneakers/worker.rb

```diff
@@ -61,11 +61,11 @@ def do_work(delivery_info, metadata, msg, handler)
           end
         rescue Timeout::Error
           res = :timeout
-          worker_error('timeout')
+          worker_error('timeout',msg)
         rescue => ex
           res = :error
           error = ex
-          worker_error('unexpected error', ex)
+          worker_error('unexpected error', msg, ex)
```

Comment:
Seems like the message for an error should go to `logger.debug` not `logger.error`. I'm thinking messages with sensitive data or it filling up logs if you have a handler that keeps retrying and getting an error.